### PR TITLE
feat: add auto-tag

### LINKED ISSUE #18: Add auto-tag

### PR DESCRIPTION
**Pull request by spaenleh**
This PR adds the auto-tag feature.

closes #18

### CODE REVIEW

File: workflow-templates/auto-tag.properties.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Auto-tag issues and PR with released version",
```
[the remark this answers is not in the document]
Comment (pyphilia) [reply workflow-templates/auto-tag.properties.json:2]: Not sure for latest tho

**Review by pyphilia — APPROVED** [review]
LGTM 👍

File: workflow-templates/auto-tag.yml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+  setup-labels:
+    if: ${{ !github.event.release }}
+    runs-on: ubuntu-latest
+    env:
+      REPO: ${{ github.event.repository.full_name }}
+    steps:
+      - name: Get `${{ env.PRE_LABEL_NAME }}` label
+        id: label-list
+        run: echo label=$(gh label list --search $PRE_LABEL_NAME -L 1 --json name -q '.[].name' --repo $REPO) >> $GITHUB_OUTPUT
```
Comment (dialexo) [inline workflow-templates/auto-tag.yml:25]: Info: what does this step do? Does it search for the label to see if it exists?

**Review by dialexo — APPROVED** [review]

File: workflow-templates/auto-tag.properties.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Auto-tag issues and pull requests",
+  "description": "Workflow to handle tagging closed issues and merged PRs with 'un-released' tag. Works in conjonction with release-please auto-tag step.",
+  "iconName": "graasp-logo",
+  "categories": ["Typescript", "Javascript"]
```
[the remark this answers is not in the document]
Comment (spaenleh) [reply workflow-templates/auto-tag.properties.json:5]: You are right, I can remove the categories

**Pull request merged**